### ISSUE TITLE
Placing of various RS blocks cause crash which prevents server restart

### ISSUE THREAD

**Issue by DeviousRex** [raoulvdberge/refinedstorage#1050]
I keep removing a ticking entity (Solderer in this case) that is causing my server to crash and preventing it from restarting.

I remove the entity using NTBExplorer, delete the region on the server, replace that region with the edited region (with TE removed) and try to restart the server. The server crashes and when I look at the crash report and region again, the TE has returned. 

I have even removed the entire chunk from the region that contains the TE, which allows the server to start but results in the loss of everything in that chunk. When I replace the now newly generated region with the previous region, the server won't start, as expected.

This happened once before (with an External Storage Entity) but with some tweaking, I was able to get the server to start again and remove the entity in-game. That method isn't working for this instance.

#### Steps to reproduce:
1. Add entity (Solderer), game crashes.
2. Edit Region (0,0) with NTBExplorer
3. Remove Entity
4. Delete Region from Server
5. Upload edited Region to server
6. Restart Server *crash*
...

#### Version (Make sure you are on the latest version before reporting):

- Minecraft: 1.11.2
- Forge: forge-1.11.2-13.20.0.2223 and forge-1.11.2-13.20.0.2228
- Refined Storage: 1.4.1

Does this issue occur on a server? Yes

#### If a (crash)log is relevant for this issue, link it here:

[crash-2017-03-11_14.54.33-server.txt](https://github.com/raoulvdberge/refinedstorage/files/836196/crash-2017-03-11_14.54.33-server.txt)

**Comment by DeviousRex** [comment]
I wound up fixing the issue by rolling back to an earlier backup, but am still interested if anyone has seen what I have described and if there is a fix.

Thanks

**Comment by raoulvdberge** [comment]
That is weird. Open data/refinedstorage_nodes.dat and remove the problematic node (at pos (220,57,315) from your crashreport).

**Comment by DeviousRex** [comment]
Thank you very much.

**Comment by DeviousRex** [comment]
Issue happened again. This time I placed a Crafting Monitor in the same location (220,57,315). As soon as I placed it, the server crashed.

I went into data/refinedstorage_nodes.dat and removed the Crafting Monitor and restarted the server. It crashed before finishing the restart (same issue as before).

I went into data/refinedstorage_nodes.dat again and confirmed that the Crafting Monitor was not there and not not reappeared; it had not. 

I went into the region file (0,0) and found the crafting monitor there. I deleted it, saved the file and tried to restart the server. It crashed before restarting.

I checked the region file and the crafting monitor was back. I deleted it again with the same result (crash while starting.

I have attached a new crash report.

Thanks

[crash-2017-03-13_18.53.23-server.txt](https://github.com/raoulvdberge/refinedstorage/files/840065/crash-2017-03-13_18.53.23-server.txt)
